### DISCUSSION TITLE
Set decimal place dynamically in q-input

### DISCUSSION THREAD

**Discussion by akashramtohul4** [quasarframework/quasar#9527] [CLI - PWA mode]
Hello friends 

i have a q-input  which accept decimal place. but i want to set the decimal place dynamically .
is it still use  mask="#.##"?

Can someone please Help

Thanks

**Comment by ontwikkelfabriek** [comment]
> want to set the decimal place dynamically

Do you mean with `set decimal place`  the number of decimals in a q-input? Or do you mean where the decimal is rendered in the q-input?

**Comment by akashramtohul4** [comment]
Sorry for not being clear 😅

That I mean is the number of decimal in the q-input

For example the q-input allow a number with 2 decimal  now I want to change
it to allow 3 decimal

Can you please help

**Reply by ontwikkelfabriek** [reply]
@akashramtohul4 

Here's a basic demo:
https://codepen.io/ontwikkelfabriek/pen/KKWQowy

**Comment by akashramtohul4** [comment]
Thanks it worked

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=icon>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=link>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
